src/storage.rs

```diff
@@ -19,24 +19,15 @@ use std::{collections::HashMap, fmt::Debug};
 
 #[derive(Clone, Copy, Debug, Serialize)]
 #[serde(tag = "Main")]
-pub(crate) enum StorableType {
+pub(crate) enum MainStorageType {
```

Comment:
suggestion: Maybe `PersistentStorageType`? `Main` doesn't really mean anything to me. Not a huge deal since I think this will go away soonish.